src/Juvix/Bohm.hs

```diff
@@ -3,10 +3,21 @@ module Juvix.Bohm
     module Juvix.Bohm.Translation,
     module Juvix.Bohm.Type,
     module Juvix.Bohm.Default,
+    erasedCoreToBohm,
   )
 where
 
 import Juvix.Bohm.Default
 import Juvix.Bohm.Parser
 import Juvix.Bohm.Translation
 import Juvix.Bohm.Type
+import qualified Juvix.Core.Erased.Types as Erased
+import Juvix.Library
+
+erasedCoreToBohm ∷ ∀ primVal. Erased.Term primVal → Bohm
+erasedCoreToBohm term =
+  case term of
+    Erased.Var s → Symbol' s
+    Erased.Prim _ → undefined
```

Comment:
What does Prim present here? If there is nothing we can change this to safely, let's make the type `Either ??? Bohm`

Reply:
In the follow-up PR (#158) I will fix this so that it is complete.